src/components/project/services.ts

```diff
@@ -0,0 +1,31 @@
+import Vue from 'vue';
```

Comment:
Get rid of services and make the axios calls in the components like we spoke about.

Reply:
Done